bundles/org.openhab.binding.freeathomesystem/README.md

```diff
@@ -0,0 +1,81 @@
+# ABB/Busch-free@home Smart Home binding
+
+ openHAB ABB/Busch-free@home binding based on the offical free@home local api
+
+![alt text](https://github.com/jannodeluxe/jannnnoooo/blob/main/free_at_home_logo_1.jpg)
```

Comment:
The logo is added automatically to the readme, if you upload it to the openHAB website. See https://www.openhab.org/docs/developer/bindings/#add-your-binding-s-logo-to-the-openhab-website

Reply:
OK, I understand. This is here only some illustration for the text.

Reply:
logo removed, image of the free@home devices are kept as illustration. is it OK?